ios/fastlane/Fastfile

```diff
@@ -62,13 +72,22 @@ platform :ios do
     if options[:submit]
       deploy_testflight 
     end
+    increment_build_number(build_number:options[:ci_build_num])
```

Comment:
I don't see a reason to sync build numbers with those sent to TestFlight at this time.